scripts/measures/2018/import-ia-measures.js

```diff
@@ -0,0 +1,79 @@
+const parse = require('csv-parse/lib/sync');
+const _ = require('lodash');
+const fs = require('fs');
+const path = require('path');
+
+const Constants = require('../../../constants.js');
+/**
+ * `import-ia-measures` reads an IA CSV file and creates valid measures,
+ * then writes the resulting json to a staging measures-data-ia.js file.
+ */
+
+const IA_CSV_COLUMN_NAMES = {
+  'Activity Name': 'title',
+  'Activity Description': 'description',
+  'Activity ID': 'measureId',
+  'metricType': 'metricType',
+  'firstPerformanceYear': 'firstPerformanceYear',
+  'lastPerformanceYear': 'lastPerformanceYear',
+  'weight': 'weight',
+  'subcategoryId': 'subcategoryId',
+  'cehrtEligible': 'cehrtEligible'
+};
+
+// Accounts for TRUE, True, true, X, x...
+// and people sometimes insert extra spaces
+function cleanInput(input) {
+  return input.trim().toLowerCase();
+}
+
+// map specific csv input values to their representation in the measures schema
+function mapInput(rawInput) {
+  const input = cleanInput(rawInput);
+  if (input === '' || input === 'null') {
+    return null;
+  } else if (input === 'true') {
+    return true;
+  } else if (input === 'false') {
+    return false;
+  } else if (Constants.validPerformanceYears.includes(Number(input))) {
+    return Number(input);
+  } else {
+    return rawInput.trim();
+  }
+}
+
+/**
+ * [convertCsvToMeasures description]
+ * @param  {array of objects}  each object in the array represents
+ * a new measure (row)
+ * @return {array}            Returns an array of measures objects
+ *
+ * Note: We trim all data sourced from CSVs because people sometimes unintentionally
+ * include spaces or linebreaks
+ */
+function convertIACSVsToMeasures(iaCSVRows) {
```

Comment:
Do we have a convention of lowercasing all letters after the first even when it's an acronym? I thought so but maybe it's just me...

Reply:
Fixed in aa7759ee0607c8d3b7b58d07a0e0c8c0164b906b!